recipe/meta.yaml

```diff
@@ -20,6 +21,8 @@ source:
 build:
   number: {{ build }}
   skip: true  # [win]
+  skip: true  # [linux and cuda_compiler_version != "11.8"]
```

Comment:
This ensures Linux only builds with CUDA and only used CUDA 11.8 (no other versions)